pkg/network/ovn_kubernetes.go

```diff
@@ -268,6 +268,9 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data.Data["OVNIPsecDaemonsetEnable"] = OVNIPsecDaemonsetEnable
 	data.Data["OVNIPsecEnable"] = OVNIPsecEnable
 
+	// Set progressing to true until IPsec DaemonSet is rendered when EW IPsec config is enabled.
```

Comment:
@pperiyasamy can you add a TODO here explaining what we want to do in the future and/or pointing to the jira issue? thanks!

Reply:
Done. Thanks a lot @jcaamano and @kyrtapz for a quick offline reviews on this PR.